uthash.yaml

```diff
@@ -0,0 +1,33 @@
+package:
```

Comment:
Similar to above, doesn't look to be actively maintained (~1 year). We might want to discuss whether moving this to private packages is a better option given this

Reply:
This is also a simple header-only data structure library.  Its low risk to have.